packages/edit-post/src/components/layout/style.scss

```diff
@@ -12,6 +12,11 @@
 	}
 }
 
+
+.edit-post-layout__content {
+	overflow-x: hidden;
```

Comment:
I don't think we should add this. While we haven't seen it yet, there may be cases where you need to be able to scroll horizontally in the layout content. Most of the time the horizontal scrollbar is helpful to discover accidental horizontal scrollbars, i.e. when content is too wide, but you could even imagine a horizontally scrolling theme! :) https://wordpress.com/theme/shelf